server.js

```diff
@@ -1,33 +1,117 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+
+// My choosen dataset
+import books from './data/books.json';
 
 // Defines the port the app will run on. Defaults to 8080, but can be 
 // overridden when starting the server. For example:
-//
+
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+const port = process.env.PORT || 8080;
+const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+  res.send('Hello, welcome to Gabriella´s API with books! Possible endpoints to use: /books (for seeing all books), /books/book/:id, (to search by Book ID) /books/author/:author (to search by author) and /books/top-rated (to see the 20 highest rated books) With filter and query parameter you can find search =high to get 50 books with high rating and you can use page=pagenumber to se 50 per page where it exist 10 pages in total')
+});
+
+// The first endpoint, returns a collection of books
+app.get('/books', (req, res) => {
+  res.json(books)
+});
+
+// Filter query
+app.get('/books/filter', (req, res) => {
```

Comment:
This could instead be a query under the `/books` endpoint so that a query can look like this:
`/books?average_rating&page=1`
instead of
`/books/filter...`